docs/source/modules.rst

```diff
@@ -217,7 +217,12 @@ analyses/mlenabledcgeslc
 
 analyses/montecarlofailureprobability
 =====================================
-..  autoclass:: montecarlofailureprobability.montecarlofailureprobability.MonteCarloFailureProbability
+.. deprecated:: 1.19.0
+   This class will be deprecated soon. Use :class:`montecarlolimitstateprobability.MonteCarloLimitStateProbability` instead.
+
+analyses/montecarlolimitstateprobability
+=====================================
```

Comment:
can you match the number of "=" in line 224 the same as the number of characters in line 223?

Reply:
also, I just remembered the above things, so let's also check this again for the other ones after we create a release branch using the release branch

Reply:
I matched everything in the module.rst and will do another check on release branch when it's there.